frontend/javascripts/admin/voxelytics/log_tab.tsx

```diff
@@ -143,23 +144,34 @@ export default function LogTab({
   runId,
   taskName,
   isRunning,
+  beginTime,
+  endTime,
 }: {
   workflowHash: string;
   runId: string;
   taskName: string;
   isRunning: boolean;
+  beginTime: Date | null;
+  endTime: Date | null;
 }) {
-  const [showTimestamps, setShowTimestamps] = useState(false);
+  const [showTimestamps, setShowTimestamps] = useState(true);
   const [isFullscreen, setIsFullscreen] = useState(false);
   const [isLoading, setIsLoading] = useState(true);
-  const [level, setLevel] = useState("DEBUG");
+  const [level, setLevel] = useState(LOG_LEVELS.INFO);
```

Comment:
Did this work? On the deployed dev instance, the default log level still seems to be `DEBUG`.

Reply:
I just redeployed the dev instance.